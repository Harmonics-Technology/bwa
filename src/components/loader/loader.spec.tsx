import { render } from "test";

import Loader from "./loader";

describe("Loader component tests", () => {
  it("renders without crashing", () => {
    const view = render(<Loader />);

    expect(view).toBeTruthy();
  });

  it("snapshot test", () => {
    const view = render(<Loader />);

    expect(view).toMatchSnapshot();
  });
});
