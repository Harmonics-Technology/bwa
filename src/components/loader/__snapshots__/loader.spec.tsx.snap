// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component tests snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <li>
        <div
          class=" animate-pulse"
          role="status"
        >
          <span
            class="sr-only"
          >
            Loading...
          </span>
          <div
            class="flex items-center px-4 py-4 sm:px-6"
          >
            <div
              class="flex min-w-0 flex-1 items-center"
            >
              <div
                class="min-w-0 flex-1 px-4 md:grid md:grid-cols-2 md:gap-4"
              >
                <div>
                  <div
                    class="h-2.5 bg-gray-200 rounded-full dark:bg-gray-200 w-48 mb-4"
                  />
                  <p
                    class="mt-2 flex items-center text-sm text-gray-500"
                  />
                  <div
                    class="h-2 bg-gray-200 rounded-full dark:bg-gray-200 max-w-[360px]"
                  />
                </div>
                <div
                  class="hidden md:block"
                >
                  <div>
                    <div
                      class="h-2 bg-gray-200 rounded-full  max-w-[360px]"
                    />
                  </div>
                  <div
                    class="h-2 bg-gray-200 rounded-full  max-w-[360px] mt-6"
                  />
                </div>
              </div>
            </div>
            <div>
              <svg
                aria-hidden="true"
                class="h-5 w-5 text-green-400"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M7.21 14.77a.75.75 0 01.02-1.06L11.168 10 7.23 6.29a.75.75 0 111.04-1.08l4.5 4.25a.75.75 0 010 1.08l-4.5 4.25a.75.75 0 01-1.06-.02z"
                  fill-rule="evenodd"
                />
              </svg>
            </div>
          </div>
        </div>
      </li>
    </div>
  </body>,
  "container": <div>
    <li>
      <div
        class=" animate-pulse"
        role="status"
      >
        <span
          class="sr-only"
        >
          Loading...
        </span>
        <div
          class="flex items-center px-4 py-4 sm:px-6"
        >
          <div
            class="flex min-w-0 flex-1 items-center"
          >
            <div
              class="min-w-0 flex-1 px-4 md:grid md:grid-cols-2 md:gap-4"
            >
              <div>
                <div
                  class="h-2.5 bg-gray-200 rounded-full dark:bg-gray-200 w-48 mb-4"
                />
                <p
                  class="mt-2 flex items-center text-sm text-gray-500"
                />
                <div
                  class="h-2 bg-gray-200 rounded-full dark:bg-gray-200 max-w-[360px]"
                />
              </div>
              <div
                class="hidden md:block"
              >
                <div>
                  <div
                    class="h-2 bg-gray-200 rounded-full  max-w-[360px]"
                  />
                </div>
                <div
                  class="h-2 bg-gray-200 rounded-full  max-w-[360px] mt-6"
                />
              </div>
            </div>
          </div>
          <div>
            <svg
              aria-hidden="true"
              class="h-5 w-5 text-green-400"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M7.21 14.77a.75.75 0 01.02-1.06L11.168 10 7.23 6.29a.75.75 0 111.04-1.08l4.5 4.25a.75.75 0 010 1.08l-4.5 4.25a.75.75 0 01-1.06-.02z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </li>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
